common/flatpak-dir.c

```diff
@@ -1891,7 +1891,10 @@ extra_data_progress_report (guint64 downloaded_bytes,
 static gboolean
 flatpak_dir_setup_extra_data (FlatpakDir           *self,
                               OstreeRepo           *repo,
+                              const char           *repository,
+                              const char           *ref,
                               const char           *rev,
+                              GPtrArray            *subdirs_arg,
```

Comment:
There is no need to pass in the subdirs when you only want to get the commits anyway, they will never be used.